src/modules/leaderboard.ts

```diff
@@ -0,0 +1,118 @@
+import { Sequelize, Op } from "sequelize";
+
+import db from "src/db";
+import logger from "src/utils/logger-handler";
+
+async function updateLeaderboardRow(address: string, property: string, value: number) {
+  const userLeaderboard = await db.leaderboard.findOne({
+    where: { address }
+  });
+
+  if(userLeaderboard) {
+    userLeaderboard[property] = value;
+
+    await userLeaderboard.save();
+  } else
+    await db.leaderboard.create({
+      address,
+      [property]: value,
+    });
+}
+
+/**
+ * Update leaderboard bounties quantity. If the parameter is not passed it will count all bounties.
+ */
+async function updateLeaderboardBounties(state?: "canceled" | "closed") {
+  try {
+    const bountiesOfCreators = await db.issues.findAll({
```

Comment:
`findAndCountAll()` should work as well if you want the count of results, no?

Reply:
@moshmage the response of `findAndCountAll()` when grouping is more complex than the response of using `COUNT` function

`findAndCountAll`
![image](https://user-images.githubusercontent.com/9003126/211567913-b9a21ff9-827c-469a-9105-5f131040bc40.png)

`COUNT`
![image](https://user-images.githubusercontent.com/9003126/211568349-0d24b921-7657-48bd-b032-06a1f5dc6aac.png)